src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -21,6 +21,7 @@ Microsoft.CodeAnalysis.INamedTypeSymbol.ExtensionMarkerName.get -> string?
 Microsoft.CodeAnalysis.INamedTypeSymbol.ExtensionParameter.get -> Microsoft.CodeAnalysis.IParameterSymbol?
 Microsoft.CodeAnalysis.INamedTypeSymbol.IsExtension.get -> bool
 Microsoft.CodeAnalysis.IPropertySymbol.ReduceExtensionMember(Microsoft.CodeAnalysis.ITypeSymbol! receiverType) -> Microsoft.CodeAnalysis.IPropertySymbol?
+Microsoft.CodeAnalysis.Operations.ICollectionExpressionOperation.ConstructArguments.get -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.IOperation!>
```

Comment:
>ConstructArguments [](http://example.com/codeflow?start=65&length=18)

Was this named settled on in API review? It's not clear from the thread

Reply:
not fully.  my goal was to have basic things like names hashed out in the PR. Rikki made a good point that we have ConstrctMethod.  So having ConstrctArguments was a sensible parallel.